docs/debugging.md

```diff
@@ -10,9 +10,9 @@ $ compile                                     # run compilation manually
 
 ## Debugging Fuzzers with GDB
 
-If you decide to debug a fuzzer with gdb (which is already installed in libfuzzer-runner image),
+If you decide to debug a fuzzer with gdb (which is already installed in base-runner image),
```

Comment:
here's the thing: gdb is not installed in base-runner. And it pulls so many .so with itself that I don't want to install it there. Hence the need for a separate image. Rather than removing this image it should probably be renamed into ossfuzz/debug-runner instead?